src/node/nodestate.h

```diff
@@ -1081,6 +1081,8 @@ namespace ccf
                                         const Service::Write& w) {
         if (w.at(0).value.status == ServiceStatus::OPEN)
         {
+          this->consensus->set_f(
+            1); // TODO: we should make f come from a KV table
```

Comment:
Maybe ticket this?

Reply:
This isn't so clear cut, f could also be ephemeral and derived by each node when it hits network open (that is in a table), based on what it has in its nodes table/list.